spec/features/admin/orders_spec.rb

```diff
@@ -12,18 +12,38 @@
     bundle.parts << [parts]
     line_item.update_attributes!(quantity: 3)
     order.reload.create_proposed_shipments
-    order.finalize! 
+    order.finalize!
   end
 
   it "allows admin to edit product bundle" do
+    change_orders_item_quantity_to 2
+
+    visit spree.edit_admin_order_path(order)
+
+    verify_3_parts_have_their_quantity_set_to 2
+  end
+
+  def change_orders_item_quantity_to(n)
```

Comment:
Second actually, since there is only one spec that is using these helper methods, do we really need them??